tajo-core/src/main/java/org/apache/tajo/master/QueryInProgress.java

```diff
@@ -163,6 +168,12 @@ public void submmitQueryToMaster() {
 
     try {
       writeLock.lockInterruptibly();
+    } catch (Exception e) {
```

Comment:
I have a question. 
Would you explain why writeLock.lockInterruptibly() is called subsequently?